core/tazama-core.sh

```
@@ -1,7 +1,7 @@
 #!/bin/bash
-# filepath: ./tazama.sh
+# filepath: ./tazama-core.sh
 
-set -e
+cd "$(dirname "$0")"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)